spec/features/archive_link_spec.rb

```diff
@@ -27,7 +27,21 @@
       login_as(user)
       visit proposals_path
 
-      expect(page).to_not have_content("View the archive")
+      expect(page).to_not have_content("View all completed")
+    end
+  end
+
+  scenario "adds archive link for canceled proposals" do
+    ClimateControl.modify CLOSED_PROPOSAL_LIMIT: "2" do
+      user = create(:user)
+      3.times.map do |i|
+        create(:proposal, requester: user, status: "canceled")
```

Comment:
can be re-written as 

`create_list(:proposal, 3, requester: user, status: "canceled")`